layouts/shortcodes/cf-launch.html

```diff
@@ -1,5 +1,5 @@
 <a
-  href="https://console.aws.amazon.com/cloudformation/home?#/stacks/quickcreate?stackName={{ .Get 1 }}&templateURL=https://s3.amazonaws.com/{{ $.Site.Params.template_bucket  }}/templates/{{ $.Site.Params.branch  }}/{{ .Get 0 }}"
+href="https://console.aws.amazon.com/cloudformation/home?#/stacks/create/review?stackName={{ .Get 0 }}&templateURL=https://s3.amazonaws.com/{{ $.Site.Params.template_bucket  }}/templates/{{ $.Site.Params.branch  }}/{{ .Get 1 }}" 
```

Comment:
Can you rebase with upstream? looks like you don't have latest changes

Reply:
done